internal/pkg/aws/cloudformation/changeset.go

```diff
@@ -163,8 +163,7 @@ func (cs *changeSet) describe() (*ChangeSetDescription, error) {
 	}, nil
 }
 
-// execute executes a created change set.
-func (cs *changeSet) execute() error {
+func (cs *changeSet) checkExecutionStatus() error {
```

Comment:
nit:
```suggestion
func (cs *changeSet) isExecutable() error {
```

Reply:
Removed the method to unbox the code! This function is no longer there